### DISCUSSION TITLE
Settings not being saved when connected over bluetooth

### DISCUSSION THREAD

**Discussion by clkish1988** [nefarius/DsHidMini#116] [Q&A] [answered]
Unable to change mode over bluetooth. Always stuck in xinput mode even after saving settings

**Comment by nefarius** [comment]
Well, that sucks.

**Comment by nefarius** [comment]
Moved to discussion since issue can't be reproduced and contained **zero useful information** to debug anyway.

**Comment by nefarius** [comment, marked as answer]
Found it, see #117
